app/views/partials/foyer/conjoint.html

```diff
@@ -6,14 +6,39 @@
         class="btn btn-default btn-block"
         btn-radio="false"
         ng-model="vitEnCouple"
-        focus-on="vitSeul"
-        ng-click="$emit('individu.conjoint', null)">Je vis seul(e)</button>
+        ng-click="$emit('individu.conjoint', null)">
+        Je vis seul(e)
```

Comment:
Unify masculine/feminine distinctions to dots rather than parens (`.e`), more readable.